src/stirling/source_connectors/socket_tracer/uprobe_manager.h

```diff
@@ -628,6 +629,7 @@ class UProbeManager {
   //               Without clean-up, these could consume more-and-more memory.
   absl::flat_hash_set<std::string> openssl_probed_binaries_;
   absl::flat_hash_set<std::string> scanned_binaries_;
+  std::vector<std::string> uprobe_opt_out_;
```

Comment:
I'd make this an `absl::flat_hash_set` because you're going to be doing lookups.

Reply:
Completed in https://github.com/pixie-io/pixie/pull/1971/commits/1207906d583090c861d06971e20a6da9c38d69fd